software/CONFIGURATION.md

```diff
@@ -42,7 +42,7 @@ default configuration:
 # Experimental configuration
 
 Other configuration properties are used by [experimental features](software/firmware/experimental/__init__.py)
```

Comment:
This link is broken, I think because this file is already within the `software` folder